src/otx/engine/engine.py

```diff
@@ -516,24 +518,48 @@ def optimize(
 
         Example:
             >>> engine.optimize(
+            ...     checkpoint=<checkpoint/path>,
             ...     datamodule=OTXDataModule(),
             ...     checkpoint=<checkpoint/path>,
             ... )
         CLI Usage:
             To optimize a model, run
                 ```python
                 otx optimize
+                    --checkpoint <CKPT_PATH, str>
                     --model <CONFIG | CLASS_PATH_OR_NAME> --data_root <DATASET_PATH, str>
                     --model.model_name=<PATH_TO_IR_XML, str>
                 ```
         """
+        if checkpoint is not None:
+            checkpoint = str(checkpoint)
+        elif self.checkpoint is not None:
+            checkpoint = str(self.checkpoint)
+        else:
+            checkpoint = None
```

Comment:
```suggestion
        checkpoint = checkpoint if checkpoint is not None else self.checkpoint
```

Reply:
It seems that checkpoint should be inserted with .xml or .onnx suffix. Can check point be none?

Reply:
Updated